packages/lib-classifier/src/components/Classifier/components/TaskArea/TaskArea.js

```diff
@@ -65,7 +65,6 @@ export default function TaskArea({
         flex
       >
         <Tab
-          disabled={disabled}
```

Comment:
Grommet warns that an active tab can't be disabled, so I've removed this to fix that warning.